cloud-data-protection/src/services/base/serviceBase.tsx

```diff
@@ -0,0 +1,22 @@
+export abstract class ServiceBase {
+    protected onError(error: any) {
+        if (!error?.response?.data) {
+            return ServiceBase.unknownError();
+        }
+
+        // 401 unauthorized
+        if (error.response.data.statusDescription && error.response.data.message) {
+            return Promise.reject(error.response.data.statusDescription + ': ' + error.response.data.message.toLowerCase());
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        return Promise.reject(error.response.data.statusDescription + ": " + error.response.data.message.toLowerCase());
```